guides/source/caching_with_rails.md

```diff
@@ -718,25 +886,6 @@ You can also set the strong ETag directly on the response.
 response.strong_etag = response.body # => "618bbc92e2d35ea1945008b42799b0e7"
 ```
 
-Caching in Development
-----------------------
-
-By default, caching is *enabled* in development mode with
-[`:memory_store`](#activesupport-cache-memorystore).
-This doesn't apply to Action Controller caching, which is disabled
-by default.
-
-To enable Action Controller caching Rails provides the `bin/rails dev:cache` command.
-
-```bash
-$ bin/rails dev:cache
-Development mode is now being cached.
-$ bin/rails dev:cache
-Development mode is no longer being cached.
-```
-
-To disable caching set `cache_store` to [`:null_store`](#activesupport-cache-nullstore)
-
 References
```

Comment:
The links mentioned here are 12 years old. Maybe we can remove them? Or integrate the first one in this guide later on?

Reply:
I removed them, I haven't seen "references" as a common section in other guides as well.